### DISCUSSION TITLE
workflows  not working

### DISCUSSION THREAD

**Discussion by muzaffer72** [SagerNet/SagerNet#511] [Q&A]
I have build error. I used github workflows. Whats problem.

![resim](https://user-images.githubusercontent.com/10040551/182714810-41cf5ffb-4e32-487f-b14e-31e3fa48b7a1.png)

**Comment by reeceyng** [comment]
Me too. But I successfully build it locally.

**Reply by muzaffer72** [reply]
I can't even build locally. Can you help with this.
